full_import.sh

```diff
@@ -1,9 +1,24 @@
 #!/bin/bash
 
-until host ${1} &> /dev/null
+# add some variables around retries
+retryCount=0
+maxRetries=15
+retryCountDelay=20
+
+# run the script as sudo so that you
+if [ -z "$SUDO_COMMAND" ]
+then
+  echo -e "Only root can run this script.\nRelaunching script with sudo.\n"
+  sudo -E $0 $*
+  exit 0
+fi
```

Comment:
Relaunch the script with sudo so it does not prompt you for the sudo password partway down the script.